From d360a22e7a0b75f1e081f6d7cbf4d55745ecce6b Mon Sep 17 00:00:00 2001
From: Sakib Sajal <sakib.sajal@windriver.com>
Date: Mon, 27 May 2019 17:57:04 -0400
Subject: [PATCH] Bash: Allow ptest, new-exp, to be run as root

Upstream-Status: inappropriate [embedded specific]

The ptest was failing as it was expecting non-root user.
Ptests are only run as root.

Signed-off-by: Sakib Sajal <sakib.sajal@windriver.com>
Signed-off-by: Randy Macleod <randy.macleod@windriver.com>
---
 tests/new-exp.right | 2 +-
 tests/new-exp.tests | 4 ----
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/tests/new-exp.right b/tests/new-exp.right
index c0f78fbb..9a42ec81 100644
--- a/tests/new-exp.right
+++ b/tests/new-exp.right
@@ -624,7 +624,7 @@ c Sub = 0 2 4 8
 <'ab cd'> 
 <'4'> <'ab cd'> 
 <> 
-argv[1] = <host(2)[5.0]$ >
+argv[1] = <host(2)[5.0]# >
 < 	
 >
 <' \t\n'>
diff --git a/tests/new-exp.tests b/tests/new-exp.tests
index 557d6b44..31e15468 100644
--- a/tests/new-exp.tests
+++ b/tests/new-exp.tests
@@ -1,7 +1,3 @@
-if (( $UID == 0 )); then
-	echo "new-exp.tests: the test suite should not be run as root" >&2
-fi
-
 # must do this because posix mode causes process substitution to be disabled
 # and flagged as a syntax error, which causes the shell to exit
 set +o posix
-- 
2.20.1

